### PR TITLE
Make the move UI replacable/extensible

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

File: src/main/java/com/cloudbees/hudson/plugins/folder/relocate/DefaultRelocationUI.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package com.cloudbees.hudson.plugins.folder.relocate;
```
Comment (KostyaSha) [inline src/main/java/com/cloudbees/hudson/plugins/folder/relocate/DefaultRelocationUI.java]: maybe hudson->jenkins?

**Comment by varmenise** [comment]
:bee:

File: src/main/java/com/cloudbees/hudson/plugins/folder/relocate/DefaultRelocationUI.java (hunk 1 above)
Comment (stephenc) [inline src/main/java/com/cloudbees/hudson/plugins/folder/relocate/DefaultRelocationUI.java]: I do not see the value in making that move for just one class

**Comment by stephenc** [comment]
@reviewbybees done

File: src/main/java/com/cloudbees/hudson/plugins/folder/relocate/DefaultRelocationUI.java (hunk 1 above)
Comment (KostyaSha) [inline src/main/java/com/cloudbees/hudson/plugins/folder/relocate/DefaultRelocationUI.java]: ok

**Comment by stephenc** [comment]
as plugin maintainer I am merging this change

**Pull request merged**